python/tank/descriptor/io_descriptor/appstore.py

```diff
@@ -672,7 +673,13 @@ def __get_app_store_proxy_setting(self):
 
         :returns: The http proxy connection string.
         """
-        config_data = shotgun.get_associated_sg_config_data()
+        try:
+            config_data = shotgun.get_associated_sg_config_data()
+        except UnresolvableCoreConfigurationError:
+            # This core is not part of a pipeline configuration, we're probably bootstrapping,
+            # so skip the check and simply return the regular proxy settings.
+            return self._sg_connection.config.raw_http_proxy
```

Comment:
I suggest emitting a debug log message at this point just to highlight that we are not using the ini file. (Yay logging!)